### ISSUE TITLE
This function is not used, wet objects cannot be dried by baking

### ISSUE THREAD

**Issue by wangluodaxia** [Phazorknight/Cogito#350]
![image](https://github.com/user-attachments/assets/6729bf94-96f9-4f2d-afb1-c4e55179cccd)

**Comment by Phazorknight** [comment]
The systemic properties are still very much WIP. 
Since the the dynamic property checks aren't working as I'd want them, I've put the additional property changes on the backburner.

**Comment by Phazorknight** [comment]
@wangluodaxia 
agreed. The naming of current properties, statuses, and functions is currently confusing. 
Even when first writing it, I struggled.
Updating this to a new, consistent naming scheme should be part of a future update.
